std_msgs/Float64 LSS_Probability
std_msgs/Float64 DS_Probability
std_msgs/Float64 RSS_Probability
